cli/cmd/compliance_aws.go

```diff
@@ -39,46 +39,14 @@ var (
 		Long:    `List all AWS accounts configured in your account.`,
 		Args:    cobra.NoArgs,
 		RunE: func(_ *cobra.Command, _ []string) error {
+			cli.StartProgress(" Fetching compliance information...")
```

Comment:
nit: spinner text is a little generic. Could make it say 'fetching Aws Accounts'